src/main/java/org/gridsuite/voltageinit/server/service/VoltageInitWorkerService.java

```diff
@@ -183,8 +187,12 @@ public Consumer<Message<String>> consumeRun() {
             } catch (Exception e) {
                 LOGGER.error(FAIL_MESSAGE, e);
                 if (!(e instanceof CancellationException)) {
+                    Map<String, String> errorIndicator = new HashMap<>();
+                    errorIndicator.put(ERROR_KEY, ERROR_DURING_VOLTAGE_PROFILE_INITIALISATION);
+
+                    resultRepository.insert(resultContext.getResultUuid(), new OpenReacResult(OpenReacStatus.NOT_OK, Collections.emptyList(), errorIndicator));
+                    resultRepository.insertStatus(List.of(resultContext.getResultUuid()), OpenReacStatus.NOT_OK.name());
```

Comment:
Maybe we could keep the NOT_OK for the case the solver doest not find any solution.
And we could use a FAILED status here.
But it wouldn't be homogeneous with the instruction below...

Reply:
Actually it wouldn't change anything. We'll see if we do it later